job.js

```diff
@@ -92,20 +120,23 @@ class BigQueryJob {
      * @return {Object}
      */
     getQueryOptions(opts) {
-        return {
+        let options = {
             // This params CAN be overridden
             dryRun: false,
             useLegacySql: false,
             query: this.getQuerySQL(),
-
+            destination: this.destinationTable,  // With destination = null, BigQuery will generate a temporal table that lives for 24 hs
```

Comment:
I would add a `this.getDestionationTable()` ,

You can do:
`...this.getDestionationTable()` and it can return the writeDisposition as well.

Reply:
By default it should return `destination: null`